app-shell/src/python/index.ts

```diff
@@ -0,0 +1,72 @@
+import path from 'path'
+import fs from 'fs-extra'
+import execa from 'execa'
+
+import { createLogger } from '../log'
+import { getConfig } from '../config'
+
+const log = createLogger('python')
+
+function findPython(): string | undefined {
+  const pathToPythonOverride: string | null = getConfig('python')
+    .pathToPythonOverride
+
+  let possiblePythonPaths = [
+    path.join(process.resourcesPath, 'python'),
+    path.join(process.resourcesPath, 'python', 'bin', 'python3'),
+  ]
+
+  if (pathToPythonOverride != null) {
+    possiblePythonPaths = [
+      pathToPythonOverride,
+      path.join(pathToPythonOverride, 'bin/python3'),
+      ...possiblePythonPaths,
+    ]
+  }
+
+  if (process.env.NODE_ENV !== 'production') {
+    possiblePythonPaths = [
+      ...possiblePythonPaths,
+      path.join(__dirname, '../python/bin/python3'),
+    ]
+  }
+
+  for (const path of possiblePythonPaths) {
+    try {
+      fs.accessSync(path, fs.constants.X_OK)
+      const stats = fs.statSync(path)
+      if (stats.isFile()) {
+        log.debug('Python candidate selected', { path })
+        return path
+      }
+    } catch (error) {
+      log.debug('Python candidate not executable, skipping', { path, error })
+    }
+  }
+}
+
+export function runFileWithPython(
+  srcFilePath: string,
+  destFilePath: string
+): Promise<void> {
+  const pythonPath = findPython()
+
+  if (pythonPath != null) {
+    return execa(pythonPath, [
+      '-m',
+      'opentrons.cli.__init__',
```

Comment:
With #9862 merged, you can drop the `__init__`

```suggestion
      'opentrons.cli',
```